extensions/wikia/VideosModule/Modules/VideosModuleBase.class.php

```diff
@@ -1,281 +0,0 @@
-<?php
-
-namespace VideosModule\Modules;
-
-use Wikia\Cache\AsyncCache;
-use Wikia\Logger\WikiaLogger;
-
-/**
- * Class Base
- *
- * @package VideosModule\Modules
- */
-abstract class Base extends \WikiaModel {
-
-	const THUMBNAIL_WIDTH = 268;
-	const THUMBNAIL_HEIGHT = 150;
-
-	const CACHE_TTL = 43200; // 12 hours
-	const NEGATIVE_CACHE_TTL = 300; // 5 minutes
-	const CACHE_VERSION = 5;
-
-	const LIMIT = 20;
-	const SORT = 'recent';
-
-	// A label for the data source, used to construct cache keys and for logging
-	protected $source = '';
-
-	// The maximum number of videos to fetch
-	protected $limit;
-
-	// Black listed videos we never want to show in videos module
-	protected $blacklist = [];
-
-	// A two character region code for getting region specific videos
-	protected $userRegion;
-
-	// How to sort video results (some modules)
-	protected $sort = '';
-
-	// Any categories to pull videos from (some modules)
-	protected $categories = [];
-
-	// The list of videos found
-	protected $videos = [];
-
-	// Options used when calling the getVideoDetail* methods in VideoHandlerHelper
-	protected static $videoOptions = [
-		'thumbWidth'   => self::THUMBNAIL_WIDTH,
-		'thumbHeight'  => self::THUMBNAIL_HEIGHT,
-		'getThumbnail' => true,
-		'thumbOptions' => [
-			'fluid'       => true,
-			'forceSize'   => 'small',
-		],
-	];
-
-	/**
-	 * @param string $userRegion - A two character country code
-	 */
-	public function __construct( $userRegion ) {
-		parent::__construct();
-
-		$this->initializeBlacklist();
-
-		$this->userRegion = $userRegion;
-		$this->limit = static::LIMIT;
-		$this->source = $this->getSource();
-		$this->sort = static::SORT;
-	}
-
-	abstract public function getSource();
-	abstract public function getModuleVideos();
-
-	/**
-	 * Look for videos that are blacklisted in the wgVideosModuleBlackList variable on community central
-	 */
-	protected function initializeBlacklist() {
-		$communityBlacklist = \WikiFactory::getVarByName( "wgVideosModuleBlackList", \WikiFactory::COMMUNITY_CENTRAL );
-
-		if ( is_object( $communityBlacklist ) && !empty( $communityBlacklist->cv_value ) ) {
-			$this->blacklist = unserialize( $communityBlacklist->cv_value );
-		}
-	}
-
-	/**
-	 * Get a list of videos for the source defined by this class
-	 *
-	 * @return array
-	 */
-	public function getVideos() {
-		$cacheKey = $this->getCacheKey();
-
-		$asyncCache = $this->getCache();
-		$asyncCache
```

Comment:
let's check if we can kill off async-cache related stuff too

Reply:
Still used by `MediaQueryService::getTotalVideoViewsByTitle` method

Reply:
Actually, a good point :) https://wikia-inc.atlassian.net/browse/SUS-4170